src/sys/termios.rs

```diff
@@ -776,7 +776,7 @@ libc_bitflags! {
         ALTWERASE;
         IEXTEN;
         #[cfg(not(target_os = "redox"))]
-        EXTPROC;
+        EXTPROC as tcflag_t; // uclibc gets the type wrong
```

Comment:
Is this due to a bug in libc?

Reply:
Yes, it seems that `libc`'s `uclibc` module doesn't have the right type for this item.  We could accept this change here, pending a fix in `libc`, or I could try to get a `libc` change through first.  Preferences?

Reply:
Let's fix it in libc first.

Reply:
https://github.com/rust-lang/libc/pull/2592

Reply:
Just checking in here- I'm pushing through the changes in `libc` but it's a bit slowed by some larger sibling changes.  I am still definitely pursuing this PR :)